subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheProblemReportingIntegrationTest.groovy

```diff
@@ -165,16 +165,6 @@ class ConfigurationCacheProblemReportingIntegrationTest extends AbstractConfigur
     def "serialization problems are reported and fail the build by default invalidating the cache"() {
         given:
         def configurationCache = newConfigurationCacheFixture()
-        executer.beforeExecute {
```

Comment:
I think we should add the proper annotations to `prop` and `anotherProp`, in this case `@Internal`, since the test is not about testing the task inputs/outputs.